### PR TITLE
August 2022 newsletter

### PR DESCRIPTION
**Pull request by davidfischer**
## Screenshot

Screenshot 2022-09-07 at 16-56-35 EthicalAds Newsletter - August 2022

### CODE REVIEW

File: content/posts/newsletter-august-2022.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* We worked quite a bit on updating our landing pages to better explain our product positioning
+  as an open source, privacy-focused ad network that uses contextual (rather than behavioral) targeting.
+  We added more screenshots of the product and answering more questions
+  about process and reporting before they're asked.
+* We rolled out a few advances to better detect and handle ad fraud.
+  While ad fraud is part of the industry, we work very hard to minimize it from impacting our advertisers.
+  We have an upcoming blog post detailing some of the blatant tactics we've seen.
+* Also in our ad server, we added a number of reporting improvements that improved performance
+  and showed more of our metrics in graphical form.
+* On the marketing side, we did a [ShowHN post](https://news.ycombinator.com/item?id=32650878#32653071)
```
Comment (ericholscher) [inline content/posts/newsletter-august-2022.md:24]: This should probably be at the top, no? Seems like the most newsworthy. Probably even outside a bullet point!

```suggestion
* On the marketing side, we did a [Show HN post](https://news.ycombinator.com/item?id=32650878#32653071)
```

File: content/posts/newsletter-august-2022.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+## Upcoming features
+
+The major features in our upcoming roadmap:
+
+* We continued to make really good progress on our ML model for contextual targeting.
+  After contacting some of the devs, we started working with one of the folks from SpaCy on a contract basis
+  to help improve our model and there's been some good work started there
+  that I expect to continue throughout the month.
+  It always feels great to build our product while putting money directly into the hands
+  of open source devs.
```
Comment (ericholscher) [inline content/posts/newsletter-august-2022.md:68]: 💯

**Review by ericholscher — APPROVED** [review]
Looks great.

File: content/posts/newsletter-august-2022.md (hunk 1 above)
Comment (davidfischer) [reply content/posts/newsletter-august-2022.md]: I merged this suggestion but I'll also move it up.

**Pull request merged**